code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -17,7 +17,7 @@
 	var/stream_amount = 10 //the amount of reagents transfered when in stream mode.
 	amount_per_transfer_from_this = 5
 	volume = 250
-	possible_transfer_amounts = list()
+	possible_transfer_amounts = list(5,10,15,20,25,30,50,100)

```

Comment:
transfer amounts being empty was intentional, because we want attack_self() to switch spraying modes, not transfer amounts.